types/types.go

```diff
@@ -69,9 +73,10 @@ func (m *MgmtNet) UnmarshalYAML(unmarshal func(any) error) error {
 
 	type MgmtNetWithDeprecatedFields struct {
 		MgmtNetAlias         `yaml:",inline"`
-		DeprecatedIPv4Subnet string `yaml:"ipv4_subnet,omitempty" json:"ipv4_subnet,omitempty"`
-		DeprecatedIPv6Subnet string `yaml:"ipv6_subnet,omitempty" json:"ipv6_subnet,omitempty"`
+		DeprecatedIPv4Subnet string `json:"ipv4_subnet,omitempty" yaml:"ipv4_subnet,omitempty"`
```

Comment:
this came from one of the linters. obviously superficial, but now all the things are json tags first then yaml so thats nice for consistency I guess